build/Program.cs

```diff
@@ -28,15 +28,15 @@ static void Main(string[] args)
                 .Then(AddGoZip)
                 .Then(TestPreSignedArtifacts, skip: !args.Contains("--ci"))
                 .Then(CopyBinariesToSign, skip: !args.Contains("--ci"))
-                .Then(Test, skip: args.Contains("--codeql"))
+                .Then(Test)
                 .Then(GenerateSBOMManifestForZips, skip: !args.Contains("--generateSBOM"))
```

Comment:
Aside from the other changes to remove SBOM generation, I think you missed these

Reply:
Removed it. I have not removed the actual methods which will remove separately once everything is good.